spyderlib/plugins/editor.py

```diff
@@ -219,35 +223,33 @@ def setup_page(self):
                                'always_remove_trailing_spaces', default=False)
         
         analysis_group = QGroupBox(_("Analysis"))
-        pep8_url = '<a href="http://www.python.org/dev/peps/pep-0008/">PEP8</a>'
-        analysis_label = QLabel(_("<u>Note</u>: add <b>analysis:ignore</b> in "
-                                  "a comment to ignore code/style analysis "
-                                  "warnings. For more informations on style "
-                                  "guide for Python code, please refer to the "
-                                  "%s page.") % pep8_url)
-        analysis_label.setWordWrap(True)
+        pep_url = '<a href="http://www.python.org/dev/peps/pep-0008/">PEP8</a>'
+        pep8_label = QLabel(_("<i>(Refer to the {} page)</i>").format(pep_url))
+        pep8_label.setOpenExternalLinks(True)
         is_pyflakes = codeanalysis.is_pyflakes_installed()
         is_pep8 = codeanalysis.get_checker_executable('pep8') is not None
-        analysis_label.setEnabled(is_pyflakes or is_pep8)
-        pyflakes_box = newcb(_("Code analysis")+" (pyflakes)",
+        pyflakes_box = newcb(_("Real time code analysis")+" (pyflakes)",

```

Comment:
Please remove `pyflakes` from this text, and use the same exact text we have in `Optional dependencies` related to `pyflakes`, i.e.

     Real-time code analysis